charts/hlf-k8s/values.yaml

```diff
@@ -12,10 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-image:
-  repository: substrafoundation/hlf-k8s
-  tag: latest
-  pullPolicy: IfNotPresent
+images:
+  hlfk8s:
+    repository: substrafoundation/hlf-k8s
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  hlfk8sca:
```

Comment:
would `hlfk8sCa` be more readable and consistent here ? As we use lowerCamelCase everywhere else.

Reply:
Yes :)